editor/editor_property_name_processor.cpp

```diff
@@ -105,9 +105,12 @@ EditorPropertyNameProcessor::EditorPropertyNameProcessor() {
 	capitalize_string_remaps["aa"] = "AA";
 	capitalize_string_remaps["aabb"] = "AABB";
 	capitalize_string_remaps["adb"] = "ADB";
+	capitalize_string_remaps["and"] = "and"; // Stop word (don't capitalize it).
```

Comment:
Makes sense, though I'd suggest moving all the stop words to their own block. Grouping them semantically makes more sense than alphabetically together with acronyms IMO.

Reply:
This would also allow to avoid repeating the comment.